src/split-map/split-map.fork.test.ts

```diff
@@ -1,19 +1,31 @@
 import 'regenerator-runtime/runtime';
-import { createDomain, fork, serialize, allSettled } from 'effector';
+import {
+  createDomain,
+  fork,
+  allSettled,
+  createEvent,
+  createStore,
+  createApi,
+} from 'effector';
 
 import { splitMap } from './index';
 
 test('works in forked scope', async () => {
-  const app = createDomain();
-  const source = app.createEvent<{ first?: number; another?: boolean }>();
+  const $target = createStore(0);
+
+  const source = createEvent<{ first?: number; another?: boolean }>();
   const out = splitMap({
     source,
     cases: {
       first: (payload) => payload.first,
     },
+    targets: {
```

Comment:
Modifying tests while adding a new feature looks like bad practice.

Due to potentially broke backward compatibility.
Proposing to create new tests with `target` support, along with just `splitMap` without `targets`. Otherwise it can be merged only when `patronum@3` is released.